src/test/components/MenuButtons.test.js

```diff
@@ -41,9 +45,24 @@ jest.mock('../../utils/gz');
 import { shortenUrl } from '../../utils/shorten-url';
 jest.mock('../../utils/shorten-url');
 
+import { symbolicateProfile } from 'firefox-profiler/profile-logic/symbolication';
+jest.mock('firefox-profiler/profile-logic/symbolication');
+
 // Mock hash
 const hash = 'c5e53f9ab6aecef926d4be68c84f2de550e2ac2f';
 
+// We need IndexedDB to get a SymbolStore that's necessary for symbolication
+// to even start, in some of the tests for this file.
```

Comment:
I considered moving all tests for the MetaInfoPanel into their own test file. But in the patch to add the "delete" button I'll change this a bit more (I'll make the meta panel component only one part of the displayed panel) and so I would like to wait until it's finished. Maybe it will still be a good idea to move it after I made all changes.